website/src/components/SearchComponents/SearchMacro/SearchMacroDesktop/SearchMacroDesktop.jsx

```diff
@@ -0,0 +1,59 @@
+/*
+Copyright 2023 The Vitess Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React from "react";
+
+import "../SearchMacroDesktop/searchMacroDesktop.css";
+
+import { formatByteForGB, fixed } from "../../../../utils/Utils";
```

Comment:
nit: group imports together and/or leave a new line between the component definition and the last import